tests/sentry/workflow_engine/handlers/condition/test_event_frequency_handlers.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+
+from sentry.issues.grouptype import GroupCategory
+from sentry.models.group import Group
+from sentry.rules.conditions.event_frequency import ComparisonType, EventFrequencyCondition
+from sentry.testutils.skips import requires_snuba
+from sentry.workflow_engine.handlers.condition.event_frequency_handlers import (
+    EventFrequencyCountHandler,
+)
+from sentry.workflow_engine.models.data_condition import Condition
+from tests.sentry.workflow_engine.handlers.condition.test_base import (
+    ConditionTestCase,
+    EventFrequencyQueryTestBase,
+)
+
+pytestmark = [pytest.mark.sentry_metrics, requires_snuba]
+
+
+class TestEventFrequencyCountCondition(ConditionTestCase):
+    condition = Condition.EVENT_FREQUENCY_COUNT
+    rule_cls = EventFrequencyCondition
+    payload = {
+        "interval": "1h",
+        "id": EventFrequencyCondition.id,
+        "value": 1000,
+        "comparisonType": ComparisonType.COUNT,
+    }
+
+    def test_count(self):
+        dc = self.create_data_condition(
+            type=self.condition,
+            comparison={"interval": "1h", "value": 1000},
+            condition_result=True,
+        )
+
+        self.assert_slow_cond_passes(dc, 1001)
+        self.assert_slow_cond_does_not_pass(dc, 999)
```

Comment:
```suggestion
        self.assert_slow_cond_passes(dc, [1001])
        self.assert_slow_cond_does_not_pass(dc, [999])
```
^ should fix the mypy error.